src/scripts/components/Tooltip.jsx

```diff
@@ -239,8 +239,8 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   removeTooltip(tooltip) {
-    tooltip.remove();
     this.tooltipId = undefined;
+    tooltip.parentNode.removeChild(tooltip);
```

Comment:
Add comment here why we are replacing tooltip.remove(); to tooltip.parentNode.removeChild(tooltip);